Python/bytecodes.c

```diff
@@ -318,8 +315,7 @@ dummy_func(
             TO_BOOL_STR,
         };
 
-        specializing op(_SPECIALIZE_TO_BOOL, (counter/1, value -- value)) {
-            TIER_ONE_ONLY
+        tier1 specializing op(_SPECIALIZE_TO_BOOL, (counter/1, value -- value)) {
```

Comment:
`specializing` implies that it is tier 1 only, so maybe drop the `tier1` here, and have the code generator handle it?